openpype/hosts/traypublisher/plugins/publish/validate_workfile.py

```diff
@@ -6,7 +6,7 @@
 class ValidateWorkfilePath(pyblish.api.InstancePlugin):
     """Validate existence of workfile instance existence."""
 
-    label = "Collect Workfile"
+    label = "Validate Workfile"
     order = pyblish.api.ValidatorOrder - 0.49
     families = ["workfile"]
     hosts = ["traypublisher"]
```

Comment:
Noting this here even though it's somewhat unrelated to this PR.

This particular plug-in checks `sourceFilepath` in data. A quick query over the code base shows that ONLY this validator and collector use that particular `instance.data`. It doesn't appear to be used for any output, etc.

Could we somehow make to this more generic so that there's no need for this additional data key for this single use case but we could do e.g. a validator over all `instance.data[representation]` and its files? (even though most representations get added during Extractors after Validator).

Validating our input content exists seems like something we (need to do or are doing) elsewhere too so I imagine we might want to unify where possible?

Reply:
The whole logic of traypublisher will be more generic, but not in this PR.